docs/use-observable-state.md

```diff
@@ -32,43 +36,51 @@ class Clock extends ylem.Component { // 👀
 
 ## ViewModel Classes
 
-If passing state around so it can be mutated directly seems too "wild west", use ylem to define testable types that protect the state with accessors, methods, and setters.
+As your app grows in complexity, we recommend defining ViewModels for managing state and interactions. Simple ES classes can be used to isolate code in a reusable and testable manner:
 
 ```js
-class PaginateState extends ylem.Object {
-  constructor(data) {
-    super(data);
-
-    this.count = this.count || Infinity;
-    this.offset = this.offset || 0;
-    this.limit = this.limit || 10;
+// ViewModel.js
+class ViewModel {
```

Comment:
this needs to extend can-observe's `Object`, also an `ylem.Object`, just like in `@connect`.

Reply:
Also, using generic names like `ViewModel` makes debugging hard. At the least, I think this should be `AppViewModel` to match the `App` component later.

Reply:
👍